tests/test_operators.py

```diff
@@ -375,6 +399,336 @@ def test_merge(self):
         expected_merged = [{"field": "test1"}, {"field": "train1"}]
         self.compare_streams(merged, expected_merged)
 
+    def test_extract_values(self):
+        input_multi_stream = MultiStream(
+            {
+                "test": [{"field": "test1"}],
+                "validation": [{"field": "validation1"}],
+                "train": [
+                    {"field": "train1"},
```

Comment:
I suggeste make the test more interesting (and test more of the behavior). Add 10 examples with more meaningful names, not in order of frequency from the start.

fish
dog
dog
cat
dog
cat
sheep
cat
fish
shark





tain

Reply:
Good idea,  will do.